crates/edr_provider/src/requests/methods.rs

```diff
@@ -286,12 +286,10 @@ pub enum MethodInvocation {
     )]
     GetStackTraceFailuresCount(()),
     /// hardhat_impersonateAccount
-    #[serde(
```

Comment:
When using `edr_eth::serde::sequence`, it ignores the manual overrides for `[de]serialize_with`. As such, I had to remove it and add empty tuples to force the procedural macro to generate a sequence.

@agostbiro do you think this is acceptable or should I implement a custom `Address` type for the JSON-RPC layer, that uses the `deserialize_address` logic?

Reply:
Works for me 👍